server/game/cards/locations/04/isleofravens.js

```diff
@@ -0,0 +1,39 @@
+const DrawCard = require('../../../drawcard.js');
+
+class IsleOfRavens extends DrawCard {
+
+    setupCardAbilities(ability) {
+        this.action({
+            title: "Kneel to shuffle a discarded card back into its owner's deck",
+            method: 'kneel',
+        });
+    }
+
+    kneel(player) {
+        if (this.kneeled)
```

Comment:
This will (or should) violate the eslint rules, everything should have braces around it